package.json

```diff
@@ -40,14 +41,15 @@
   },
   "homepage": "https://github.com/alpheios-project/webextension#readme",
   "devDependencies": {
+    "@actions/core": "^1.2.4",
     "@auth0/auth0-spa-js": "1.8.1",
     "@babel/core": "^7.10.5",
     "@babel/plugin-transform-modules-commonjs": "^7.10.4",
     "@babel/plugin-transform-runtime": "^7.10.5",
     "@babel/preset-env": "^7.10.4",
     "@babel/register": "^7.10.5",
     "@babel/runtime": "^7.10.5",
-    "alpheios-core": "github:alpheios-project/alpheios-core",
+    "alpheios-core": "git+https://github.com/alpheios-project/alpheios-core.git",
```

Comment:
A question: is this required for the build to function or is it just a switch to the more secure protocol? I'm not familiar with this and want to understand this better

Reply:
This change is really not intentional. I had previously committed the package.json with the dependency on the qa branch as it gets updated running tagged-commit.mjs, so that the prior build was something Monica could download from github and test. I removed the qa branch from the dependency but didn't readjust the rest of the url. I think it just gets set this way because of the way the install is done 

`npm install https://github.com/alpheios-project/alpheios-core#${buildInfo.branch}`